js/graph_spectrum_plot.js

```diff
@@ -400,142 +423,152 @@ GraphSpectrumPlot._drawGridLines = function(canvasCtx, sampleRate, LEFT, TOP, WI
         canvasCtx.lineTo(i * (WIDTH / ticks), HEIGHT);
 
         canvasCtx.stroke();
-        var textAlign = (i==0)?'left':((i==ticks)?'right':'center');
-        this._drawAxisLabel(canvasCtx, (frequency.toFixed(0))+"Hz", i * (WIDTH / ticks), HEIGHT + MARGIN, textAlign);
+        let textAlign;
+        if (i === 0) {
+            textAlign = 'left';
+        } else {
+            textAlign = (i === ticks) ? 'right' : 'center';
+        }
```

Comment:
This could be done a lot more elegantly with a `switch` replacing the `if` and the remaining ternary.